cpp/src/interop/dlpack.cpp

```diff
@@ -25,11 +25,18 @@
 #include <dlpack/dlpack.h>
 
 #include <algorithm>
+#include "cudf/utilities/traits.hpp"
```

Comment:
Might one to move this up
```suggestion
#include <cudf/utilities/traits.hpp>
```